issue_release_status.go

```diff
@@ -31,12 +54,85 @@ func (handler *PRCreateHandler) Handle(ctx context.Context, eventType, deliveryI
 	installationID := githubapp.GetInstallationIDFromEvent(&event)
 
 	ctx, logger := githubapp.PreparePRContext(ctx, installationID, repository, prNum)
-
-	logger.Debug().Msgf("Event action is %s", event.GetAction())
 	if event.GetAction() != "created" {
 		return nil
 	}
 
+	prBase := event.GetPullRequest().GetBase().GetRef() // The branch which the pull request is ending.
+
+	// Retrieve auto-release-policy
+
+	serviceName := event.GetRepo().GetName()
+	servicePath := handler.releaseManagerURL + "/policies?service="
+
+	// Create client
+	client := &http.Client{}
+
+	// Set route
+	resp, err := client.Get(servicePath + serviceName)
+	if err != nil {
+		return errors.Wrap(err, "issueing a GET to specified URL")
+	}
+
+	req, err := http.NewRequest("GET", servicePath+serviceName, nil) // dublicate work??
+	if err != nil {
+		return errors.Wrap(err, "wrapping new request")
+	}
+
+	req.Header.Add("Authorization", "Bearer "+handler.releaseManagerAuthToken)
+
+	resp, err = client.Do(req)
+
+	var policyResponse ListPoliciesResponse
+
+	body, err := ioutil.ReadAll(resp.Body)
+
+	defer resp.Body.Close()
+
+	json.Unmarshal(body, &policyResponse)
```

Comment:
[`json.Unmarshal`](https://godoc.org/encoding/json#Unmarshal) returns an error in case of ... well errors 🙈 

It should be checked :)